zerver/actions/message_edit.py

```diff
@@ -901,6 +915,7 @@ def exclude_duplicates_from_subscription(
             )
 
         subscriber_ids = set(subscriptions.values_list("user_profile_id", flat=True))
+        subscriber_ids.update(old_stream_subs_not_in_new_stream)
```

Comment:
Let's not call this `subscriber_ids` -- I think `notifiable_ids` would be fine, but it's confusing to use a variable called `subscriber_ids` if it doesn't contain exactly some set of subscribers.